src/frontend/next/src/components/SearchPage/SearchResults.tsx

```diff
@@ -0,0 +1,84 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { useSWRInfinite } from 'swr';
+import { Container, Box, Grid, Typography, ListSubheader } from '@material-ui/core';
+import useSiteMetadata from '../../hooks/use-site-metadata';
+import Timeline from '../Timeline';
+import Spinner from '../Spinner';
+
+type SearchPageProps = {
+  text: string | null;
+  filter: string | null;
+};
+
+const useStyles = makeStyles(() => ({
+  spinner: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  searchResults: {
+    padding: 0,
+    width: '100%',
+    justifyContent: 'center',
+  },
+}));
+
+const SearchResults = ({ text, filter }: SearchPageProps) => {
+  const classes = useStyles();
+  const { telescopeUrl } = useSiteMetadata();
+  const prepareUrl = (index: number) =>
+    text
+      ? `${telescopeUrl}/query?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`
+      : // Should we return an empty string? or null here?
```

Comment:
Why would text ever be null?  Should we not always have it be a valid string, perhaps `''` in the case that the user hasn't done anything yet?